bungiesearch/indices.py

```diff
@@ -70,6 +72,30 @@ def get_mapping(self):
         '''
         return {'properties': dict((name, field.json()) for name, field in iteritems(self.fields))}
 
+    def collect_analysis(self):
+        '''
+        :return: a dictionary which is used to get the serialized analyzer definition from the analyzer class.
+        '''
+        analysis = {}    
+        for field in self.fields.values():
+            for analyzer_name in ('analyzer', 'index_analyzer', 'search_analyzer'):
+                if not hasattr(field, analyzer_name):
+                    continue
+
+                analyzer = getattr(field, analyzer_name)
+
+                if not isinstance(analyzer, Analyzer):
+                    continue
+
+                d = analyzer.get_analysis_definition()
```

Comment:
I'm not a huge fan of single letter variables. Could this be renamed to `definition` or `anlzr` or something a bit longer?